adapters/adorsys-integ-adapter/src/main/java/de/adorsys/xs2a/gateway/adorsys/service/provider/AdorsysIntegAccountInformationServiceProvider.java

```diff
@@ -26,8 +26,9 @@
 
 public class AdorsysIntegAccountInformationServiceProvider implements AccountInformationServiceProvider {
```

Comment:
Could you, please, rename this and the rest providers to the name convention, you used for Sparda/DAB/Consors? AdorsysIntegAccountInformationServiceProvider -> AdorsysIntegServiceProvider.
Thanks!

Reply:
AdorsysIntegAccountInformationServiceProvider provides only one service. I don't think it should be renamed.

Reply:
According to the providers' design, that was introduced by current PR, such services as `AdapterNameAccountInformationServiceProvider` and `AdapterNamePaymentInitiationServiceProvider` don't exist no more. From now on there is only `AdapterNameServiceProvider`, that provides some number of services. 
That's why, I think, that we should follow the one convention in our project, but not support both (the old and the new ones).

Reply:
Merged and renamed rest of the providers.

Reply:
Thanks!